ui/page/send/layout.go

```diff
@@ -110,19 +126,6 @@ func (pg *Page) Layout(gtx layout.Context) layout.Dimensions {
 						)
 					})
 				}),
-				layout.Stacked(func(gtx C) D {
-					if pg.moreOptionIsOpen {
-						inset := layout.Inset{
-							Top:   values.MarginPadding40,
-							Right: values.MarginPadding20,
-						}
-						return inset.Layout(gtx, func(gtx C) D {
-							border := widget.Border{Color: pg.Theme.Color.Background, CornerRadius: values.MarginPadding5, Width: values.MarginPadding1}
-							return border.Layout(gtx, pg.clearAllBtn.Layout)
```

Comment:
The border acts as a temporary shadow pending when the shadow widget/functionality is implemented.
With out this, this is what you have.
![image](https://user-images.githubusercontent.com/27733432/130740495-0c389d47-b57d-4803-94f8-332114edd22b.png)